.devcontainer/Dockerfile

```diff
@@ -48,12 +48,9 @@ RUN usermod --shell /usr/bin/zsh ${USERNAME} \
 COPY --from=nodejs /usr/local/include/node /usr/local/include/node
 COPY --from=nodejs /usr/local/lib/node_modules /usr/local/lib/node_modules
 COPY --from=nodejs /usr/local/bin /usr/local/bin
-RUN npm install --global npm@latest
 
 # zsh and commitizen (cz) configurations
 COPY ${CONFIGURATION_DIR_PATH}/.zshrc ${USERHOME}/.zshrc
 COPY ${CONFIGURATION_DIR_PATH}/.czrc ${USERHOME}/.czrc
 RUN chown ${USERNAME}.${USERNAME} ${USERHOME}/.zshrc ${USERHOME}/.czrc \
-    && chmod +x ${USERHOME}/.zshrc ${USERHOME}/.czrc
-
-ENTRYPOINT exec top -b
\ No newline at end of file
+    && chmod +x ${USERHOME}/.zshrc ${USERHOME}/.czrc
```

Comment:
if you do `chmod +x` locally and add + commit the change, this line is superfluous.

Reply:
Thanks for your feedback, done making the changes.